pages/linux/init.md

```diff
@@ -0,0 +1,23 @@
+# init
+
+> Init is a daemon process that start during booting continue to run until the system is shut down.
+
+- Shutdown the system:
```

Comment:
=> "Shut down"

Reply:
Good point! I guess because I say / write `shutdown`, I never though to think of the proper / correct form here